compiler-plugin/build.gradle

```diff
@@ -41,6 +44,9 @@ shadowJar {
     configurations = [project.configurations.compile]
     relocate 'org.jetbrains.kotlin.com.intellij', 'com.intellij'
     relocate 'org.jetbrains.kotlin.load', 'kotlin.reflect.jvm.internal.impl.load'
+
+    //Required for idea-plugin tests
+    relocate 'org.apache.log4j', 'shadow.org.apache.log4j'
```

Comment:
Thank you 😄 🎉

Reply:
@jansorg Is that the correct dependency? For running it in CI

Reply:
@i-walker The plugin is using the logging facility of the IntelliJ SDK.  IntelliJ is using log4j internally but the plugin itself isn't using log4j and log4j is in the classpath because of the dependency on the SDK.
Do you know why this is needed atm? Is there another log4j dependency which needs to be moved to work with the SDK?

Reply:
I've removed dependency between Gradle and Kotlin compiler plugin. Maybe log4j was some transitive dependency.

In any case log4j that is used in tests is binary incompatible with one that is here.